roles/ovirt-image-template/README.md

```diff
@@ -28,6 +31,18 @@ Role Variables
 | template_disk_interface | virtio           | Interface of the template disk. |
 | template_timeout   | 600                   | Amount of time to wait for the template to be created. |
 | template_nics      | {name: nic1, profile_name: ovirtmgmt, interface: virtio} | List of dictionaries that specify the NICs of template. |
+| external_templates   | UNDEF               | List of dictionaries that describes the templates |
```

Comment:
If you have a list it will be more scalable, you can have list of one disk or many, so u cover both cases.
running same role multiple times is maybe good when it done manually. 
we want to use these roles for automating things. 
I really suggest to change this role to also accept a list of qcow images ... 

i am not sure if looping over roles in the playbook is supported in the ansible.